Software 0.30.8 release notes (#2844)
astronomer/docs: software_versioned_docs/version-0.32/release-notes.md

- You can now disable Airflow and platform alerts on the Prometheus alerts dashboard by setting `prometheus.defaultAlerts.airflow.enabled` and `prometheus.defaultAlerts.airflow.enabled` to `false` in your Prometheus Helm chart. If you disable these alerts, you can still add back specific alerts or configure custom alerts using `prometheus.defaultAlerts.additionalAlerts`. See [Create custom alerts](platform-alerts.md#create-custom-alerts).
- You no longer have to set `elasticsearch.curator.age.timestring` when you configure a custom indexing pattern for [Vector logging sidecars](export-task-logs.md#export-logs-using-container-sidecars). The only required value is now `astronomer.houston.config.deployments.helm.loggingSidecar.indexPattern`. 
- You can now configure a service account specifically for your image registry using by setting `astronomer.registry.serviceaccount` in your `config.yaml` file. 
- The Kibana logging dashboard now includes a default index. 
- Added support for [Kubernetes 1.27](https://kubernetes.io/blog/2023/04/11/kubernetes-v1-27-release/).

### Bug fixes

> - Fixed an issue where if you queried a Deployment name that belonged to two different Deployments in two different Workspaces, the Houston API would retrieve the wrong Deployment. 

Author: ```suggestion
- Fixed an issue where if you queried a Deployment name that belonged to two different Deployments in two different Workspaces, the Houston API might retrieve the unintended Deployment. 
```

- Fixed an issue where Helm changes to statsd Pod resources would apply only to new Deployments. 
- Fixed an issue where data for **Disk Usage** and **Platform Overview** did not appear in Grafana. 
- Fixed an issue where you could get a 500 internal server error from the Airflow UI when switching between pages for a DAG. 
- Astronomer Software now throws an error if you attempt to install it with an unsupported version of Kubernetes.
- Removed support for Kubernetes 1.22.

> - Fixed an issue where using the Houston API to query for a Deployment that didn't exist returned an error instead of an empty set.  

Author: ```suggestion
- Fixed an issue where using the Houston API to query for a Deployment that didn't exist returned a non-descriptive error.
```
